frontend/src/modules/accounts/components/AccountEditForm.tsx

```diff
@@ -88,10 +86,6 @@ export const AccountEditForm = ({
     >
       {() => (
         <Form noValidate>
-          <FormikResetEffect
-            dependencies={[mutation]}
-            condition={mutation.isError}
-          />
```

Comment:
Removed `FormikResetEffect` component, as the Clear Input button is enough in this case. #naolenough